nss-day-cohort-60/national-parks-client-v2
# Sc events calendar

# Description
What did you do?

Added an Events page that displays a calendar with events, each park even has a different color. 

# Steps to test

1. Fetch the sc-events-calendar branch and switch to it
2. Make sure that your server is up-to-date with main, run server
3. navigate to the 'Events' tab in the client.
4.  You should see a calendar displaying all events.
5. test the event filtering, in the dropdown select a park -- see the events only for selected park 
6. Add an event, see that the event has been added to the calendar (this is the part that did not work for Shaina, but did work for John)

Please ignore Modal styling... it's not the way I want it to be.
